Packages/ChineseNumber/Sources/ChineseNumber/suzhou_number.m

```diff
@@ -0,0 +1,111 @@
+#include <string.h>
+#include <stdlib.h>
+#include "trim_zero.h"
+
+#define BUFFER_SIZE 80
+
+struct suzhou_number_struct
```

Comment:
The problem with having C strings in a struct is that it's hard to remember to free all the buffers allocated here. This is a problem one wouldn't have if this were written with `std::string`s…